### PR TITLE
Add appveyor.yml.

### LINKED ISSUE #109: Add appveyor CI and get tests passing on Windows
When I run `npm test` on Windows for this project, I get a bunch of failures. It’d be helpful if this module could be used on Windows, but I am having some issues with cleanup not working the way I expect it to which might be a platform-specific thing—I still have to investigate. But with the starting point being a lot of test cases broken, it’d be nice to get those fixed and make sure they stay fixed with the help of CI.

I see you’re already using Travis. It seems that appveyor is quite popular among OSS projects and integrating with it is similar to Travis.

### PR DESCRIPTION
**Pull request by binki**
Fixes #109.

See examples of failed builds at https://ci.appveyor.com/project/binki/node-tmp

### CODE REVIEW

**Comment by binki** [comment]
The last example build for this PR is https://ci.appveyor.com/project/binki/node-tmp/build/1.0.2 , I noticed that the other branches cause builds even though they don’t have `appveyor.yml` because I decided to do my test fix attempts on a separate PR.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# https://www.appveyor.com/docs/lang/nodejs-iojs/
+
+environment:
```
Comment (raszi) [inline appveyor.yml]: Could you please fix the line endings?

**Comment by binki** [comment]
@raszi Done, see new example build at https://ci.appveyor.com/project/binki/node-tmp/build/1.0.13

**Comment by silkentrance** [comment]
@raszi did you integrate appveyor so that it will constantly check the state of the package?

**Comment by binki** [comment]
@silkentrance it looks like it's set up and just waiting for the first commit to build: https://ci.appveyor.com/project/raszi/node-tmp

**Pull request merged**